cookbook/book.toml

```diff
@@ -1,6 +1,10 @@
 [book]
-authors = ["Harshil Jani"]
+authors = ["Toby Harding<me@tobin.cc>", "Einherjar <realeinherjar@proton.me>"]
```

Comment:
@tcharding I have no idea what to put in here...

Reply:
Leave Harshil's name in there please because he did some prior work. Definitely throw your name in there since you are working on it. Leave mine out for now please because I have not added anything meaningful, I'll add it later if/when I do. (I'm Tobin BTW :)

Reply:
Sorry I thought Hashil was a placeholder 😓